llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -836,6 +836,25 @@ def IREELinalgExt_SetEncodingOp : IREELinalgExt_PureOp<"set_encoding",[
   }];
 }
 
+def IREELinalgExt_UpperBoundTileSizeOp : IREELinalgExt_PureOp<"upper_bound_tile_size",
```

Comment:
I just landed some changes to the structure of this LinalgExtOps.td file in https://github.com/openxla/iree/pull/14388 . Before merging, could you sync and ensure that this op is in the right `OpDocGroup` / part of the file? (OpGroupDataTilingOps?)

```c++
//===----------------------------------------------------------------------===//
// Data tiling ops
//===----------------------------------------------------------------------===//

def OpGroupDataTilingOps : OpDocGroup {}

let opDocGroup = OpGroupDataTilingOps in {

// YOUR OP SHOULD BE HERE? //

} // OpGroupDataTilingOps
```

Reply:
Yup, checked, it does belong in there. Thanks!